doc/classes/Tree.xml

```diff
@@ -270,6 +270,7 @@
 			<argument index="0" name="item" type="Object">
 			</argument>
 			<description>
+				Causes the [Tree] to jump to the specified item.
```

Comment:
```suggestion
				Makes the [Tree] scroll to the specified item.
```

Also, the description should be clarified: does it scroll so that the item appears centered, or does it scroll as little as possible?